Images/index.html

```diff
@@ -0,0 +1,326 @@
+      </p>
+      <div class="centered3">LET'S CONNECT</div>
+      <div class="centered4">
+         <ul>
+            <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+            <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+            <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+            <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+            <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+         </ul>
+      </div>
+   </section>
+   <main class="mobile_main">
+      <section class="work-container">
+         <div></div>
+         <h1>Tonic</h1>
+         <div></div>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <div>
+            <ul>
+               <li><a href="#"><button>html</button></a></li>
+               <li><a href="#"><button>css</button></a></li>
+               <li><a href="#"><button>javaScript</button></a></li>
+            </ul>
+         </div>
+         <button>See Project</button>
+      </section>
+      <section class="work-container2">
+         <div></div>
+         <h1>Multi-Post Stories</h1>
+         <div></div>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <div>
+            <ul>
+               <li><a href="#"><button>html</button></a></li>
+               <li><a href="#"><button>css</button></a></li>
+               <li><a href="#"><button>javaScript</button></a></li>
+            </ul>
+         </div>
+         <button>See Project</button>
+      </section>
+      <section class="work-container3">
+         <div></div>
+         <h1>Tonic</h1>
+         <div></div>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <div>
+            <ul>
+               <li><a href="#"><button>html</button></a></li>
+               <li><a href="#"><button>css</button></a></li>
+               <li><a href="#"><button>javaScript</button></a></li>
+            </ul>
+         </div>
+         <button>See Project</button>
+      </section>
+      <section class="work-container4">
+         <div></div>
+         <h1>Multi-Post Stories</h1>
+         <div></div>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <div>
+            <ul>
+               <li><a href="#"><button>html</button></a></li>
+               <li><a href="#"><button>css</button></a></li>
+               <li><a href="#"><button>javaScript</button></a></li>
+            </ul>
+         </div>
+         <button>See Project</button>
+      </section>
+      <section class="ams">
+         <h1>About Myself</h1>
```

Comment:
- [ ] As the previous reviewer mentioned, for semantics reasons and best practices, a web page is supposed to have only one (`h1 `tag ) which is already present in the header section of your page. So kindly use one of the other (`hx` tags) `(e.g: h2, h3,...)` on the `About Myself` text instead of the `h1` tag you currently have :+1: